app/controllers/option_sets_controller.rb

```diff
@@ -9,6 +9,7 @@ class OptionSetsController < ApplicationController
   # authorization via cancan
   load_and_authorize_resource
   skip_authorization_check only: :child_nodes
+  skip_authorize_resource only: :child_nodes
```

Comment:
Why are both of these needed?

Reply:
This is not related to our change in linked option sets, and is a bugfix in your code. The description of the bug is in the issue.

@smoyth I think I've not tested putting out skip_authorization_check.